yuriechan/wasteLookup
# 10 search bar

> Related issue

#10 

> Blocker 

~~#13 #14 #15~~

> Notes to the reviewer

Styling will be handled on this issue #16 

> How does the reviewer test on local?

- `git pull` to update the local branch
- `npm start` to start the application 

> reviewer todo:

- Check to see all requirements are met as expected 
- Leave notes on code that can be improved
